.gitignore

```
@@ -82,7 +82,7 @@ playground.xcworkspace
 
 # Carthage
 # Add this line if you want to avoid checking in source code from Carthage dependencies.
-# Carthage/Checkouts
+  Carthage/Checkouts
```

お疲れ様です！OKです✨

細かいですが、他の行に合わせてスペースは削除しましょう👌

```suggestion
Carthage/Checkouts
```

次回このファイルを修正するタイミングで大丈夫です🙆‍♂️